src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -216,59 +216,59 @@
                                                    questions required-attributes))
                                              "All required dataset attributes must be set before publishing.")})])))
 
-           (when (or owner? writer?)
-             (if-not editing?
+             (when (or owner? writer?)
+               (if-not editing?
+                 [buttons/SidebarButton
+                  {:style :light :color :button-primary :margin :top
+                   :text "Edit" :icon :edit
+                   :onClick #(swap! state assoc :editing? true)}]
+                 [:div {}
+                  [buttons/SidebarButton
+                   {:style :light :color :button-primary :margin :top
+                    :text "Save" :icon :done
+                    :onClick (fn [_]
+                               (let [{:keys [success error]} ((@refs "workspace-attribute-editor") :get-attributes)
+                                     new-description ((@refs "description") :get-trimmed-text)
+                                     new-tags ((@refs "tags-autocomplete") :get-tags)]
+                                 (if error
+                                   (comps/push-error error)
+                                   (this :-save-attributes (assoc success :description new-description :tag:tags new-tags)))))}]
+                  [buttons/SidebarButton
+                   {:style :light :color :state-exception :margin :top
+                    :text "Cancel Editing" :icon :cancel
+                    :onClick #(swap! state dissoc :editing?)}]]))
+             (when-not editing?
```

Comment:
nit:indentation

Reply:
Upon investigation, the new indentations turned out to be correct so leaving them as they are.